projects/project.cpp

```diff
@@ -600,14 +600,81 @@ namespace projects {
          return true;
    }
 
+
+   bool Project::canRefine(spatial_cell::SpatialCell* cell) const {
+      return cell->sysBoundaryFlag == sysboundarytype::NOT_SYSBOUNDARY && (cell->sysBoundaryLayer == 0 || cell->sysBoundaryLayer > 2);
+   }
+
    int Project::adaptRefinement( dccrg::Dccrg<spatial_cell::SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid ) const {
-      int myRank;
+      phiprof::Timer refinesTimer {"Set refines"};
+      int myRank;       
       MPI_Comm_rank(MPI_COMM_WORLD,&myRank);
       if (myRank == MASTER_RANK) {
-         cerr << "(Project.cpp) Base class 'adaptRefinement' in " << __FILE__ << ":" << __LINE__ << " called. Function is not implemented for project." << endl;
+         std::cout << "Maximum refinement level is " << mpiGrid.mapping.get_maximum_refinement_level() << std::endl;
```

Comment:
I would advocate to write – if anything – to logfile.txt because it's hard to tie events in slurm files with logfile post-hoc. Given we have the option of capping AMR since today's merge to dev, I would suggest indeed having a mention somewhere of max allowed and max theoretical refinement levels (since we're often quite good at modifying cfgs during the run without proper archiving of the evolution of the cfg along the run. But in logfile, not in stdout.

Reply:
So we already have printouts about adapting refinement going to the logfile, let's delete this code block.